app/views/batch_imports/new.html.erb

```diff
@@ -1,6 +1,6 @@
 <% provide :page_title, curation_concern_page_title(curation_concern) %>
 <% provide :page_header do %>
-  <h1>CSV Import</h1>
+  <h1>Batch Imports</h1>
```

Comment:
@mcritchlow I've got the single `batch_import` route path working, which is a little trivial when I tried to configure for it. So just disregard my message earlier which is the result of my local configuration.